src/common/selftest/selftest_MINI.cpp

```diff
@@ -278,25 +278,16 @@ void CSelftest::log_open() {
         serial_otp = 0;
     }
     char serial[32] = "unknown";
-    const char *suffix = "";
-    switch (m_Mask) {
-    case stmFans:
+    const char *suffix = "_no_suffix";
```

Comment:
why is this initialization needed?

Reply:
for identification and testing purposes- when the further conditions do not meet, empty char does not provide any suffix

Reply:
why do you need any suffix at all in such case?